client/src/app/config.ts

```diff
@@ -0,0 +1,3 @@
+export let CONFIG = {
```

Comment:
@nirgn975 
This file should be on gitignore, please create a default file instead ...